src/index.js

```diff
@@ -71,7 +45,33 @@ function articleDiv(article, articleTemplateSelector, options) {
   }
 
   if (excerpt) {
-    excerpt.innerHTML = article.excerpt.rendered;
+    if (options.excerptLength) {
+      var originalExcerpt = article.excerpt.rendered;
```

Comment:
Do we have to write in ES5, i.e. not use let/const instead?

Reply:
I was just following the convention in the file, if we update this should we also update the rest ?

Reply:
no, ofc not, you can just use let/const in places which you wrote. But above you said there is a config that forces you to write in ES5, right?

Reply:
when you run `yarn build` it formats the `index.js` file. It is worth looking into to, but it isn't breaking anything and is blocking other work from being completed, so I think we should move forward with how it is now